WordPress/src/main/java/org/wordpress/android/ui/stats/StatsAbstractListFragment.java

```diff
@@ -131,6 +132,19 @@ public void onCreate(Bundle savedInstanceState) {
     @Override
     public void onSaveInstanceState(Bundle outState) {
         //AppLog.d(AppLog.T.STATS, this.getTag() + " > saving instance state");
+
+        // Do not serialize VolleyError, but rewrite in a simple stats Exception.
+        // VolleyErrors should be serializable, but for some reason they are not.
+        // FIX for https://github.com/wordpress-mobile/WordPress-Android/issues/2228
+        if (mDatamodels != null) {
+            for (int i=0; i < mDatamodels.length; i++) {
+                if (mDatamodels[i] != null && mDatamodels[i] instanceof VolleyError) {
+                    VolleyError currentVolleyError = (VolleyError) mDatamodels[i];
+                    mDatamodels[i] = StatsUtils.rewriteVolleyError(currentVolleyError, getString(R.string.error_refresh_stats));
```

Comment:
I don't like that much. We shouldn't pass exceptions (VolleyError or any other kind of exceptions) from a part of the app to another. I know we're going to rework `mDatamodels` and Stats models, but can't we just put `null` in `mDatamodels[i]` if an exception occurs?